src/bidiMapper/modules/input/ActionDispatcher.ts

```diff
@@ -878,3 +902,8 @@ function getRadii(
     radiusY: height ? height / 2 : 0.5,
   };
 }
+
+function splitGraphemes(value: string) {
```

Comment:
let's add some unit tests for the grapheme detection and split

Reply:
good point. Added.

Reply:
Not seeing the tests yet